lib/abci/handlers/errors/MaxRequestedItemsError.js

```diff
@@ -0,0 +1,14 @@
+class MaxRequestedItemsError extends Error {
```

Comment:
It will lead to an "internal error". So you need to throw ABCI Error with a specific code. I would recommend using the existing one - "invalid argument"